ledger/apply/apply.go

```diff
@@ -18,12 +18,23 @@ package apply
 
 import (
 	"github.com/algorand/go-algorand/config"
+	"github.com/algorand/go-algorand/crypto/stateproof"
 	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/data/bookkeeping"
+	"github.com/algorand/go-algorand/data/stateproofmsg"
 	"github.com/algorand/go-algorand/data/transactions"
 	"github.com/algorand/go-algorand/data/transactions/logic"
 	"github.com/algorand/go-algorand/ledger/ledgercore"
 )
 
+// StateProofs allows fetching and updating state-proofs state on the ledger
```

Comment:
Could these go into the Balances interface, since they are going to be used by apply and are implemented by cow?

Reply:
ehh maybe it's fine to be defined separately from Balances. however I don't think you need to worry about wrapping cow up into different names based on the interface name to pass it through.